src/main/java/com/google/graphgeckos/dashboard/datatypes/BuildInfo.java

```diff
@@ -52,14 +52,14 @@
 
   @Field(name = "builders")
   @Unindexed
-  private final List<Builder> builders;
+  private final List<BuildBotData> builders;
 
   /**
    * Converts a {@link #ParsedGitData ParsedGitData} object to a BuildInfo object.
```

Comment:
Update comment to reflect name change from ParsedGitData to GitHubData.

Reply:
Done!